create B827EBFFF0678CD.json

```diff
@@ -0,0 +1,18 @@
++{
```

Comment:
Please check the file content, it looks like you copy&pasted from another change, and all the plus signs got in the way. Correct the content and I will merge.